src/main/java/core/basesyntax/model/Transaction.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.model;
+
+public class Transaction {
+    private Operation operation;
+    private String name;
```

Comment:
to add a bit of context about what this transaction is, and do not forget about the getters and setters renaming
```suggestion
    private String fruitName;
```